src/QuantumSdk/Sdk/Sdk.targets

```diff
@@ -40,11 +40,11 @@
     <Delete Files="@(FilesToRemove)" />
   </Target>  
 
-  <!-- Invokes the Q# command line compiler to build the project. -->
-  <Target Name="QsharpCompile" 
+  <!-- Creates the CommandArgsFile with all the parameters for the Q# compiler. -->
+  <Target Name="PrepareQsharpCompile" 
           Condition="'$(DesignTimeBuild)' != 'true'"
           DependsOnTargets="ResolveAssemblyReferences;ResolveQscReferences;ResolveTargetPackage;BeforeQsharpCompile;_CopyFilesMarkedCopyLocal" 
```

Comment:
I wonder if it makes more sense if BeforeQsharpCompile runs before PrepareQsharpCompile or after. 
To given context: The BeforeQsharpCompile is just a target that is rooted at a particular spot in the build process for the sole purpose of making it easy for people to plug in at that point - like BeforeCompile, and like BeforeCompile it does nothing by default.